### PR TITLE
fix: update test snapshot

### PR DESCRIPTION
**Pull request by fa93hws**
Nothing but running `yarn test --updateSnapshot`....

### CODE REVIEW

File: components/table/__tests__/__snapshots__/Table.expand.test.js.snap [hunk 1]
```diff
@@ -57,7 +57,7 @@ exports[`Table.expand click to expand 1`] = `
                 </tr>
                 <tr
                   class="ant-table-row ant-table-row-level-1"
-                  data-row-key="2"
+                  data-row-key="1"
```
Comment (fa93hws) [inline components/table/__tests__/__snapshots__/Table.expand.test.js.snap:60]: What's the reason for these changes? I saw the CI on master is green. Do you have the same problem when running `yarn test` locally?

**Comment by fa93hws** [comment]
Ohhhh test fails.. 
Is that possible to be due to the dependencies mismatch since there is not lockfile..

**Comment by afc163** [comment]
Please remove node_modules and lock files and reinstall again.

**Comment by fa93hws** [comment]
> Please remove node_modules and lock files and reinstall again.

Thanks, 😞 It's actually due to the lack of the lockfile..

**Pull request closed without merging**